website/src/models/user.js

```diff
@@ -390,6 +390,15 @@ var UserSchema = new Schema({
       hatchPet: {type: Boolean, 'default': false},
       raisePet: {type: Boolean, 'default': false},
       streak: {type: Boolean, 'default': false}
+    },
+    improvementCategories: {
```

Comment:
Jeah. Array of strings? What would the best way be to still restrict what strings can be inserted?